repos/system_upgrade/common/actors/checksaphana/libraries/checksaphana.py

```diff
@@ -160,12 +184,10 @@ def _sp_rev_patchlevel_check(instance, patchlevels):
 
 def _fullfills_hana_min_version(instance):
     """ Performs a check whether the version of SAP HANA fulfills the minimal requirements for the target RHEL """
-    if version.get_target_major_version() == '8':
+    if version.matches_target_version('8.6'):
```

Comment:
if this gets called for 7.9 to 8.7+ (I know 8.7 isn't a thing) it would put in the 9.0 string always, is that what you want here?

Reply:
no, because we can line 185 only if target_version <=8.6, see lines 200-203